### PR TITLE
add login logo

### PR DESCRIPTION
**Pull request by Vedant-Tyagi**

### CODE REVIEW

**Comment by rachittyagi** [comment]
do not push settings.json, always make sure to commit the files which are related to the particular card or work

**Review by rachittyagi — CHANGES_REQUESTED** [review]

File: index.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
                             href="#portfolio">Moments</a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded"
                             href="#about">About</a></li>
+                            <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded"
+                                href="#about">Login</a></li>
```
[the remark this answers is not in the document]
Comment (Vedant-Tyagi) [reply index.html]: I had made the changes that you said me to do

[the remark this answers is not in the document]
Comment (Vedant-Tyagi) [reply index.html]: removed the live server

File: index.html [hunk 2]
```diff
@@ -36,6 +36,8 @@
                             href="#portfolio">Moments</a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded"
                             href="#about">About</a></li>
+                            <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded"
```
Comment (yaneshtyagi) [inline index.html:39]: @Vedant-Tyagi What is the breakpoint for lg-* class?

**Review by yaneshtyagi — DISMISSED** [review]

**Review by yaneshtyagi — APPROVED** [review]

**Review by yaneshtyagi — DISMISSED** [review]

**Review by yaneshtyagi — APPROVED** [review]

**Review by yaneshtyagi — APPROVED** [review]

**Pull request merged**